sources/packages/web/src/components/generic/HeaderNavigator.vue

```diff
@@ -4,7 +4,7 @@
       <slot name="title">
         <div v-if="routeLocation" class="header-title">
           <a @click="goBack()">
-            <font-awesome-icon :icon="['fas', 'arrow-left']" class="mr-2" />
+            <v-icon icon="fa:fa fa-arrow-left" size="25"></v-icon>
```

Comment:
Is the `size="25"` still needed?

Reply:
without the size, the icon is slightly big 
![image](https://user-images.githubusercontent.com/77353155/182659769-7a707297-1809-4481-a005-91c7b57337ea.png)
in Figma its slightly small,
![image](https://user-images.githubusercontent.com/77353155/182660016-a8499fa3-6c31-4f8b-8601-c5f4d21ee63c.png)